### PR TITLE
Add GDPR Compliance

### PR DESCRIPTION
**Pull request by clementdaubeuf**

### CODE REVIEW

File: ps_emailsubscription.php [hunk 1]
```diff
@@ -98,6 +98,9 @@ public function install()
                     'actionCustomerAccountAdd',
                     'additionalCustomerFormFields',
                     'displayAdminCustomersForm',
+                    'registerGDPRConsent',
```
Comment (Quetzacoalt91) [inline ps_emailsubscription.php]: Do you really need this hook `registerGDPRConsent`?

**Comment by Quetzacoalt91** [comment]
Hi there,

Can you please provide an upgrade file as well for merchants who already installed the module?
http://devdocs.prestashop.com/1.7/modules/creation/enabling_auto_update/

Thanks!

File: ps_emailsubscription.php (hunk 1 above)
Comment (clementdaubeuf) [reply ps_emailsubscription.php]: Yes, if this hook is not present the module will not be registered in the official gdpr module

**Comment by eternoendless** [comment]
Thank you @clementdaubeuf

**Pull request merged**